sdk/programmableconnectivity/azure-programmableconnectivity/setup.py

```diff
@@ -0,0 +1,70 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# Code generated by Microsoft (R) Python Code Generator.
+# Changes may cause incorrect behavior and will be lost if the code is regenerated.
+# --------------------------------------------------------------------------
+# coding: utf-8
+
+import os
+import re
+from setuptools import setup, find_packages
+
+
+PACKAGE_NAME = "azure-programmableconnectivity"
```

Comment:
Please work with architects to make sure you get approval for the package name.

Reply:
The package name will be `azure-communications-programmableconnectivity`. Although it's not in the org of Azure Communications Services  (ACS), it's been agreed that APC should be in the `Communication` namespace.

I am having discussions as to whether I should be moving the SDK folder into `communication/` or not.